### PR TITLE
Update ebpf dashboard

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
This PR is bringing changes on dashboard that will fix part of issue https://github.com/netdata/netdata/issues/11439 and it is the first step to integrate `eBPF.plugin` and `cgroup` plugin.

##### Component Name
ebpf.plugin
dashboard.plugin
##### Test Plan

1 - Compile this PR
2 - Reload your dashboard and verify the links are working.

##### Additional Information
It is not necessary to compile on different kernels, because we are only modifying dashboard.

### CODE REVIEW

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -1061,6 +1072,10 @@ netdataDashboard.context = {
         info: 'System swap memory usage. Swap space is used when the amount of physical memory (RAM) is full. When the system needs more memory resources and the RAM is full, inactive pages in memory are moved to the swap space (usually a disk, a disk partition or a file).'
     },
 
+    'system.swapcalls': {
+        info: 'Monitor calls for functions <code>swap_readpage</code>code> and <code>swap_writepage</code>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows swap access per <a href="#menu_apps_submenu_swap">application</a>.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:1076]:
```suggestion
        info: 'Monitor calls to functions <code>swap_readpage</code> and <code>swap_writepage</code>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows swap access per <a href="#menu_apps_submenu_swap">application</a>.'
```

File: web/gui/dashboard_info.js [hunk 2]
```diff
@@ -808,6 +808,16 @@ netdataDashboard.submenu = {
         info: 'Statistics of the peer variables for each peer configured in <code>/etc/ntp.conf</code> as shown by the readvar billboard <code>ntpq -c "rv &lt;association&gt;"</code>, while each peer is assigned a nonzero association ID as shown by <code>ntpq -c "apeers"</code>. The module periodically scans for new/changed peers (default: every 60s). <b>ntpd</b> selects the best possible peer from the available peers to synchronize the clock. A minimum of at least 3 peers is required to properly identify the best possible peer.'
     },
 
+    'mem.page_cache': {
+        title: 'page cache (eBPF)',
+        info: 'Monitor calls for functions used to manipulate <a href="https://learn.netdata.cloud/docs/agent/collectors/ebpf.plugin#memory" target="_blank">Linux page cache</a>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows page cache manipulation per <a href="#menu_apps_submenu_page_cache">application</a>.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:813]:
```suggestion
        info: 'Monitor calls to functions used to manipulate <a href="https://learn.netdata.cloud/docs/agent/collectors/ebpf.plugin#memory" target="_blank">Linux page cache</a>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows page cache manipulation per <a href="#menu_apps_submenu_page_cache">application</a>.'
```

Comment (vlvkobal) [reply web/gui/dashboard_info.js:813]: The link to `Linux page cache` looks like a link to an explanation of the concept, but it isn't. Our documentation lists charts but doesn't explain anything about the page cache. Shouldn't it point to something like `https://en.wikipedia.org/wiki/Page_cache`?

File: web/gui/dashboard_info.js [hunk 3]
```diff
@@ -808,6 +808,16 @@ netdataDashboard.submenu = {
...
+    },
+
+    'apps.page_cache': {
+        title: 'page cache (eBPF)',
+        info: 'Netdata also gives a summary for these charts in <a href="#menu_mem_submenu_page_cache">Memory submenu</a>.'
+    },
+
     'filesystem.vfs': {
         title: 'vfs (eBPF)',
         info: 'Monitor calls for functions used to manipulate <a href="https://learn.netdata.cloud/docs/agent/collectors/ebpf.plugin#vfs" target="_blank">File Systems</a>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows Virtual File System per <a href="#menu_apps_submenu_vfs">application</a>.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:823]: It would also be nice to change `calls for functions` to `calls to functions` everywhere.

**Review by UmanShahzad — APPROVED** [review]

**Comment by UmanShahzad** [comment]
(Approval after @vlvkobal changes are addressed)

File: web/gui/dashboard_info.js (hunk 2 above)
Comment (thiagoftsm) [reply web/gui/dashboard_info.js]: Good suggestion, I am changing the link now, and I will return for our documentation after I update it. :handshake:

File: web/gui/dashboard_info.js (hunk 3 above)
Comment (thiagoftsm) [reply web/gui/dashboard_info.js]: Done!

```bash
bash-5.1$ grep "calls for function" web/gui/dashboard_info.js 
bash-5.1$ grep "call for function" web/gui/dashboard_info.js 
bash-5.1$ grep "calls to function" web/gui/dashboard_info.js 
        info: 'Monitor calls to functions used to manipulate <a href="https://en.wikipedia.org/wiki/Page_cache" target="_blank">Linux page cache</a>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows page cache manipulation per <a href="#menu_apps_submenu_page_cache">application</a>.'
        info: 'Monitor calls to functions used to manipulate <a href="https://learn.netdata.cloud/docs/agent/collectors/ebpf.plugin#vfs" target="_blank">File Systems</a>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows Virtual File System per <a href="#menu_apps_submenu_vfs">application</a>.'
        info: 'Successful or failed calls to functions <code>tcp_sendmsg</code>, <code>tcp_cleanup_rbuf</code>, and <code>tcp_close</code>.'
        info: 'Failed calls to functions <code>tcp_sendmsg</code>, <code>tcp_cleanup_rbuf</code>, and <code>tcp_close</code>.'
        info: 'Successful or failed calls to functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>.'
        info: 'Failed calls to functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>.'
        info: 'Successful or failed calls to functions <code>vfs_read</code> and <code>vfs_write</code>. This chart may not show all file system events if it uses other functions to store data on disk.'
        info: 'Failed calls to functions <code>vfs_read</code> and <code>vfs_write</code>.'
        info: 'Successful or failed calls to functions <code>vfs_fsync</code>.'
        info: 'Failed calls to functions <code>vfs_fsync</code>.'
        info: 'Successful or failed calls to functions <code>vfs_open</code>.'
        info: 'Failed calls to functions <code>vfs_open</code>.'
        info: 'Successful or failed calls to functions <code>vfs_create</code>.'
        info: 'Failed calls to functions <code>vfs_create</code>.'
```

**Comment by thiagoftsm** [comment]
> (Approval after @vlvkobal changes are addressed)

Everything was  addressed. :handshake:

**Review by vlvkobal — APPROVED** [review]

File: web/gui/dashboard_info.js [hunk 4]
```diff
@@ -1061,6 +1072,10 @@ netdataDashboard.context = {
         info: 'System swap memory usage. Swap space is used when the amount of physical memory (RAM) is full. When the system needs more memory resources and the RAM is full, inactive pages in memory are moved to the swap space (usually a disk, a disk partition or a file).'
     },
 
+    'system.swapcalls': {
+        info: 'Monitor calls tor functions <code>swap_readpage</code> and <code>swap_writepage</code>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows swap access per <a href="#menu_apps_submenu_swap">application</a>.'
```
Comment (ilyam8) [inline web/gui/dashboard_info.js:1076]:
```suggestion
        info: 'Monitor calls to functions <code>swap_readpage</code> and <code>swap_writepage</code>. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows swap access per <a href="#menu_apps_submenu_swap">application</a>.'
```

Comment (thiagoftsm) [reply web/gui/dashboard_info.js]: Thanks!

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**